binding/dotnet/Cobra/Cobra.cs

```diff
@@ -187,6 +214,36 @@ public float Process(short[] pcm)
             return voiceProbability;
         }
 
+        /// <summary>
+        /// Lists all available devices that Cobra can use for inference.
+        /// Each entry in the list can be used as the `device` argument when initializing Cobra.
+        /// </summary>
+        /// <returns>Array of all available devices that Cobra can use for inference.</returns>
+        public string[] ListHardwareDevices()
```

Comment:
follow the other PRs. This should be a static function and called `GetAvailableDevices()`. I know it's annoying that we use different names in the SDKs, but we should stay consistent